### PR TITLE
DEPR: internals

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by mroeschke** [comment]
The minimum version failures look real due to fastparquet. Maybe means we need to bump it?

**Comment by jbrockmendel** [comment]
fastparquet bump did the trick

File: ci/deps/actions-38-minimum_versions.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies:
   - blosc=1.21.0
   - bottleneck=1.3.2
   - brotlipy=0.7.0
-  - fastparquet=0.4.0
+  - fastparquet=0.6.3
```
Comment (mroeschke) [inline ci/deps/actions-38-minimum_versions.yaml:28]: If you could follow up with changing this in `install.rst` that would be great.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @jbrockmendel

File: ci/deps/actions-38-minimum_versions.yaml (hunk 1 above)
Comment (jbrockmendel) [reply ci/deps/actions-38-minimum_versions.yaml:28]: will do

**Pull request merged**